app/assets/v2/js/pages/bounty_details.js

```diff
@@ -337,12 +340,30 @@ window.addEventListener('load', function() {
                     actions.push(entry);
                     if(result['status']=='open' && !isBountyOwner(result) ){
                         var entry = {
-                            href: '/funding/claim?source='+result['github_url'],
+                            href: '/_github/auth?redirect_uri=/funding/claim?source=' + result['github_url'],
                             text: 'Claim Issue',
                             parent: 'right_actions',
                             color: 'darkBlue'
                         }
                         actions.push(entry);
+
+                        if (is_interested) {
+                            var entry = {
+                                href: '/uninterested',
+                                text: 'Remove Interest',
+                                parent: 'right_actions',
```

Comment:
two things

1. it'd be great if 'interested profiles updated AJAX'ily when i added / removed my interest
2. not sure how much thought was put into the design here.. ( i can ask someone who does design to look into it if we want).. but maybe for now we can just remove the list bullet?

Reply:
Good point. I didn't add any method of updating the view with the current user after they express interest...  Sorry about that.